source/unicode.txt

```diff
@@ -6,7 +6,7 @@ and GTK+ and discusses common errors that arise when working with strings.
 
 Definitions
 -----------
-Conceptional, a string is a list of characters such as
+Conceptually, a string is a list of characters such as
```

Comment:
There was a previous report (#103) that introduced this change, but I think you are correct in reverting it.